source/2019-08-30-the-ember-times-issue-113.md

```diff
@@ -1,32 +1,35 @@
 ---
 title: The Ember Times - Issue No. 113
-author: the crowd
+author: Jessica Jordan, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2019
 alias : "blog/2019/08/30-the-ember-times-issue-113.html"
 responsive: true
 ---
 
 Tach Emberistas! 🐹
 
+This week:
+brand-new content in the Ember Guides 🤓,
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Check Out New Content in the Official Ember Guides 🐹🤓](https://guides.emberjs.com/release/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The most popular learning resource for early and advanced Ember developers,
+the [Guides](https://guides.emberjs.com/release/), is waiting up for you with some **brand-new content**!
+
+The Guides now feature a [new section](https://guides.emberjs.com/release/accessibility/) entirely dedicated to **accessibility**. It teaches you how to create your own accessibility strategy for your application, and how to implement keyboard accessible, screenreader accessible and overall barrier-free user interfaces with Ember. If you haven't done so, be sure to [check it out today](https://guides.emberjs.com/release/accessibility/) and leave either feedback or support for improving this section in the [#topic-a11y channel](https://discordapp.com/channels/480462759797063690/480523441082990612) on the [Ember Discord](https://discordapp.com/invite/zT3asNS)!
+
+Additionally, a soon-to-be-released version of the Guides will shine with [a revamp of the testing guidelines](https://guides.emberjs.com/release/testing/). In particular, the introduction to testing has been updated and a new section explaining the different types of tests in Ember apps has been [added](https://github.com/ember-learn/guides-source/pull/964). Many thanks to both [@ijlee2](https://github.com/ijlee2) for working on the update and [@toddjordan](https://github.com/toddjordan) for the detailed review and suggestions!
```

Comment:
Yes that sounds like a good plan - the url will likely not exist forever, but I agree that it will still be useful to have it linked as part of this post since the page will still be up for quite a while. I'll update it accordingly!